### PR TITLE
VETTING: update text for vetting by phone

### PR DESCRIPTION
**Pull request by eunjuhuss**
#### Description:
Updated description text for vetting by phone

- För dig som har ett **svenskt** telefonabonnemang i ditt eget namn
- For you with a **swedish** phone number registered in your name 

----
- Swedish
Screenshot 2020-10-14 at 10 52 54
- English
Screenshot 2020-10-14 at 10 55 42

----

#### For reviewer:
- [x] Read the above description
- [x] Reviewed the code changes
- [x] Navigate to the branch (pulled the latest version)
- [x] Run the code and been able to execute the expected function
- [x] Check any styling on both desktop and mobile sizes

### CODE REVIEW

File: src/login/translation/defaultMessages/userProfile.js [hunk 1]
```diff
@@ -153,7 +153,7 @@ export const userProfile = {
   "verify-identity.vetting_phone_tagline": (
     <FormattedMessage
       id="verify-identity.vetting_phone_tagline"
-      defaultMessage={`For you with a phone number registered in your name`}
+      defaultMessage={`For you with a swedish phone number registered in your name`}
```
Comment (njons) [inline src/login/translation/defaultMessages/userProfile.js:156]: Please change to "Swedish" with capital S.

File: src/login/translation/languages/en.json [hunk 2]
```diff
@@ -395,7 +395,7 @@
   "verify-identity.verified_page-description": "The below id number is now connected to this eduID. Use your eduID to log in to services related to higher education.",
   "verify-identity.verified_pw_reset_extra_security": "Add a phone number or a security key to your eduID to keep your identity at password reset.",
   "verify-identity.vetting_freja_tagline": "For you able to create a Freja eID by visiting one of the authorised agents",
-  "verify-identity.vetting_phone_tagline": "For you with a phone number registered in your name",
+  "verify-identity.vetting_phone_tagline": "For you with a swedish phone number registered in your name",
```
Comment (njons) [inline src/login/translation/languages/en.json:398]: Please change to "Swedish" with capital S.

**Review by njons — COMMENTED** [review]
👍  just a small spelling error.

**Comment by eunjuhuss** [comment]
Thanks! I changed to capital S

**Review by njons — APPROVED** [review]
👍

**Pull request merged**